ui/widgets/selectable.js

```diff
@@ -80,8 +81,13 @@ return $.widget( "ui.selectable", $.ui.mouse, {
 					unselecting: $this.hasClass( "ui-unselecting" )
 				} );
 			} );
+			if (!this.refreshed) {
```

Comment:
Thanks. Agreed but wasn't sure if it was the style guide or not to favor an extra conditional or an extra assignment. Will remove.